fixed_bed_doe.ipynb

```diff
@@ -0,0 +1,1592 @@
+      "  72 -1.2168558e+01 4.10e+00 1.86e+12  -1.0 1.73e-03  15.0 1.00e+00 1.00e+00h  1\n",
+      "  73 -1.2168554e+01 4.63e+00 1.86e+12  -1.0 6.51e-04  15.5 1.00e+00 1.00e+00h  1\n",
+      "  74 -1.2168542e+01 4.63e+00 1.87e+12  -1.0 1.96e-03  15.0 1.00e+00 1.00e+00h  1\n",
+      "  75 -1.2168537e+01 4.11e+00 1.87e+12  -1.0 7.35e-04  15.4 1.00e+00 1.00e+00h  1\n",
+      "  76 -1.2168522e+01 2.65e+00 1.87e+12  -1.0 2.21e-03  14.9 1.00e+00 1.00e+00h  1\n",
+      "  77 -1.2168516e+01 2.49e+00 1.87e+12  -1.0 8.30e-04  15.4 1.00e+00 1.00e+00h  1\n",
+      "  78 -1.2168498e+01 1.95e+00 1.88e+12  -1.0 2.50e-03  14.9 1.00e+00 1.00e+00h  1\n",
+      "  79 -1.2168491e+01 1.87e+00 1.88e+12  -1.0 9.38e-04  15.3 1.00e+00 1.00e+00h  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "  80 -1.2168468e+01 2.31e+00 1.89e+12  -1.0 2.82e-03  14.8 1.00e+00 1.00e+00h  1\n",
+      "  81 -1.2168460e+01 2.08e+00 1.89e+12  -1.0 1.06e-03  15.3 1.00e+00 1.00e+00h  1\n",
+      "  82r-1.2168460e+01 2.08e+00 1.00e+03  -0.1 0.00e+00  14.8 0.00e+00 2.48e-10R  2\n",
+      "  83r-1.2166792e+01 6.39e+02 1.00e+03  -0.1 7.94e+02    -  7.57e-03 9.90e-04f  1\n",
+      "  84r-1.2166424e+01 9.48e+02 9.93e+02  -0.1 4.32e+01    -  1.52e-02 6.63e-03f  1\n",
+      "  85r-1.2166267e+01 9.86e+02 9.77e+02  -0.1 4.25e+00    -  1.32e-02 2.34e-02f  1\n",
+      "  86r-1.2166313e+01 9.84e+02 7.83e+02  -0.1 3.34e+00    -  1.90e-01 3.04e-02f  1\n",
+      "  87r-1.2166818e+01 1.44e+03 3.24e+03  -0.1 2.91e+00    -  2.89e-01 2.44e-01f  1\n",
+      "  88r-1.2167920e+01 2.19e+03 6.11e+04  -0.1 3.18e+00    -  4.12e-01 7.63e-01f  1\n",
+      "  89r-1.2167920e+01 2.05e+03 5.68e+04  -0.1 1.05e+00   4.0 7.38e-02 6.52e-02h  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "  90r-1.2167919e+01 1.52e+03 4.01e+04  -0.1 9.91e-01   3.5 1.35e-02 2.75e-01f  1\n",
+      "  91r-1.2167919e+01 8.38e+02 1.60e+05  -0.1 6.94e-01   3.9 4.65e-02 4.96e-01h  1\n",
+      "  92r-1.2167919e+01 8.19e+02 1.57e+05  -0.1 1.78e+00   4.4 2.14e-02 2.22e-02h  1\n",
+      "  93r-1.2167919e+01 8.05e+02 1.57e+05  -0.1 1.65e+00   4.8 2.37e-03 1.70e-02h  1\n",
+      "  94r-1.2167919e+01 7.96e+02 1.55e+05  -0.1 1.86e+00   4.3 1.10e-02 1.15e-02h  1\n",
+      "  95r-1.2167919e+01 7.72e+02 1.53e+05  -0.1 1.42e+00   4.7 7.48e-03 3.01e-02h  1\n",
+      "  96r-1.2167919e+01 7.58e+02 1.53e+05  -0.1 1.05e+00   5.2 8.94e-03 1.84e-02h  1\n",
+      "  97r-1.2167919e+01 7.53e+02 1.52e+05  -0.1 3.35e+00   4.7 5.10e-03 6.64e-03h  1\n",
+      "  98r-1.2167919e+01 7.41e+02 1.75e+05  -0.1 1.06e+00   5.1 8.54e-04 1.62e-02h  1\n",
+      "  99r-1.2167919e+01 7.36e+02 1.79e+05  -0.1 1.03e+00   5.6 5.90e-03 7.19e-03h  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      " 100r-1.2167919e+01 7.30e+02 1.75e+05  -0.1 1.97e+00   5.1 4.85e-04 7.64e-03h  1\n",
```

Comment:
We should debug all of these restoration failures. You can stop Ipopt at iteration 100 and look at the most infeasible equations.